packages/web-components/src/badge/badge.stories.ts

```diff
@@ -0,0 +1,126 @@
+import { html, when } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import type { Badge as FluentBadge } from './badge.js';
+import { BadgeAppearance, BadgeColor, BadgeShape, BadgeSize } from './badge.options';
```

Comment:
But obviously, build fails on these :/
```suggestion
import { BadgeAppearance, BadgeColor, BadgeShape, BadgeSize } from './badge.options.js';
```